bindings/python/pydoc-markdown.yml

```diff
@@ -7,7 +7,7 @@ processors:
   - type: crossref
 renderer:
   type: docusaurus
-  docs_base_path: ../../../documentation/docs/references/
+  docs_base_path: ../../documentation/sdk/docs/references/
   relative_output_path: python
```

Comment:
The client one had `references` here, do you know the difference?

Reply:
It's just because the structure for client and wallet is different